pkgs/development/misc/rappel/default.nix

```diff
@@ -0,0 +1,34 @@
+{ clang
+, fetchFromGitHub
+, libedit
+, stdenv
+}:
+
+stdenv.mkDerivation rec {
+  name = "rappel";
+
+  src = fetchFromGitHub {
+    owner = "yrp604";
+    repo = "rappel";
+    rev = "95a776f850cf6a7c21923a2100b605408ef038de";
+    sha256 = "0fmd15xa6hswh3x48av4g1sf6rncbiinbj7gbw1ffvqsbcfnsgcr";
+  };
+
+  buildInputs = [ libedit ];
+  nativeBuildInputs = [ clang ];
```

Comment:
this is not needed, `stdenv` provides gcc; which, albeit warnings, seems to work

Reply:
I made a PR to clean up some of the errors printed out by -Wpedantic: https://github.com/yrp604/rappel/pull/19